locale/exchangecalendar/en-US/delegate-calendar-dialog.dtd

```diff
@@ -1,10 +1,10 @@
  
 <!ENTITY  label.delegateCalendar.cancelbutton "Cancel">
 <!ENTITY  label.delegateCalendar.userEmail "Email">
-<!ENTITY  label.delegateCalendar.deliveryMeetingRerquestControl "Deliver Meeting Requests">
+<!ENTITY  label.delegateCalendar.deliveryMeetingRerquestControl "Deliver meeting requests">
 
-<!ENTITY  menuitem.delegateCalendar.deliverMeetingRequestslist.delegateOnly "Delegates Only">
-<!ENTITY  menuitem.delegateCalendar.deliverMeetingRequestslist.delegateAndMe "Delegates and Me">
+<!ENTITY  menuitem.delegateCalendar.deliverMeetingRequestslist.delegateOnly "Delegates only">
```

Comment:
Menu elements typically use title capitalization so this is not a good change to make.